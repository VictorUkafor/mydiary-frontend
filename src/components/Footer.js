import React from 'react';

export const Footer = () => (
  <footer>
    <div className="p copyright">
                Copyright (c) 2018 &nbsp; MyDiary &nbsp; All Rights Reserved &nbsp; |  &nbsp;
                  Designed & Developed by Victor Ukafor
    </div>
  </footer>
);


export default Footer;
