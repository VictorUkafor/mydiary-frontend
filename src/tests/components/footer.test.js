import React from 'react';
import { render, cleanup } from 'react-testing-library';
import Footer from '../../components/Footer';

afterEach(cleanup);

test('<Footer/>', () => {
  const { container } = render(<Footer />);
  expect(container.firstChild).toMatchSnapshot();
});