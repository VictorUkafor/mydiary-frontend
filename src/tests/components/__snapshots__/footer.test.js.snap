// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> 1`] = `
<footer>
  <div
    class="p copyright"
  >
    Copyright (c) 2018   MyDiary   All Rights Reserved   |    Designed & Developed by Victor Ukafor
  </div>
</footer>
`;
